src/cart/crt_iv.c

```diff
@@ -2887,16 +2888,16 @@ bulk_update_transfer_done_aux(const struct crt_bulk_cb_info *info)
 			   cb_info->buc_user_priv);
 
 	rc = crt_bulk_free(cb_info->buc_bulk_hdl);
+	output->rc = rc;
+
+	crt_reply_send(info->bci_bulk_desc->bd_rpc);
+	RPC_PUB_DECREF(info->bci_bulk_desc->bd_rpc);
 
 	if (update_cb_info) {
 		IVNS_DECREF(update_cb_info->uci_ivns_internal);
 		D_FREE(update_cb_info);
 	}
 
-	output->rc = rc;
-
-	crt_reply_send(info->bci_bulk_desc->bd_rpc);
-	RPC_PUB_DECREF(info->bci_bulk_desc->bd_rpc);
```

Comment:
just curious is it must needed to move these 3 lines to above L2894?

Reply:
Hmm. Maybe you right. Thanks.

Reply:
Ohh. No. We should send the result of bulk free back. So, I cannot free after send result about how it's complete.